qiskit/circuit/measure.py

```diff
@@ -20,6 +20,8 @@
 class Measure(Instruction):
     """Quantum measurement in the computational basis."""
 
+    opaque = True
```

Comment:
What does it mean that `measure` is opaque?

Reply:
I'm tagging anything that is not defined in terms of other more fundamental gates as opaque. This is using the terminology introduced in OpenQASM, and just means this is not "decomposeable". The unroller knows that it has reached the end of recursion once it hits these ops.